src/components/AddItem/AddItem.js

```diff
@@ -1,7 +1,113 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { firestore } from '../../lib/firebase.js';
 
 const AddItem = () => {
-  return <div>Add an Item</div>;
+  const [groceryItem, setGroceryItem] = useState('');
+  const [purchaseDate, setPurchaseDate] = useState(null);
+  const [howSoon, setHowSoon] = useState(null);
+  const listData = firestore.collection('groceryItems');
+
+  const itemInputChange = (e) => {
+    setGroceryItem(e.target.value);
+  };
+
+  const dateInputChange = (e) => {
+    setPurchaseDate(e.target.value);
+  };
+  const radioInputChange = (e) => {
+    setHowSoon(parseInt(e.target.value));
+    console.log(howSoon);
+  };
+
+  const handleSubmitClick = async (e) => {
+    e.preventDefault();
+    await listData.add({
+      // writing a new document to firestore with the select values/fields
+      name: groceryItem,
+      frequency: howSoon,
+      token: 'token',
```

Comment:
```suggestion
      token: window.localStorage.getItem('token'),
```

Reply:
...or we could pass the token down as props since Taryn and I will be storing token state in App.js. So then token would be set to `props.token` here. And then in the App.js, we would need to pass the token as a props to the AddItem component.